### DISCUSSION TITLE
iFrame Option Aufrechterhalten ?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2533] [General]
v3.2.0-alpha.42
Die Option "Aufrechterhalten" im Iframe soll doch das iFrame halten, so dass nicht bei TB-umschalten immer wieder neu geladen wird , oder?
Das funktioniert mit v3.2.0-alpha.42 nicht mehr

**Comment by Zefau** [comment]
https://github.com/Zefau/ioBroker.jarvis/issues/2537
